extensions/dacpac/src/wizard/pages/deployPlanPage.ts

```diff
@@ -99,7 +99,8 @@ export class DeployPlanPage extends DacFxConfigPage {
 			columns: this.getTableColumns(result.dataLossAlerts.size > 0),
 			width: 875,
 			height: 300,
-			ariaRole: 'alert'
+			ariaRole: 'alert',
+			moveFocusOutWithTab: true
```

Comment:
added this to follow same pattern as other tables in DacFx - @alanrenmsft et me know if we should remove this and only depend on ctrl + end

Reply:
let's remove this and rely on the common keyboard navigation.

Reply:
This feels like something we should just straight up remove then, right? (not even sure anywhere else uses this)

Reply:
I dont want to remove the property and implementation* yet since its in azdata and also because I want to wait and see if someone from accessibility team comes back with older guidance. Will remove the usage for now.